cellprofiler/module.py

```diff
@@ -1034,3 +1035,226 @@ def visible_settings(self):
             self.x_name,
             self.y_name
         ]
+
+
+class ObjectProcessing(Module):
+    category = "Object Processing"
+
+    def add_measurements(self, workspace):
+        objects = workspace.object_set.get_objects(self.y_name.value)
+
+        labels = objects.segmented
+
+        if not objects.volumetric:
+            labels = numpy.asarray([labels])
+
+        centers = mahotas.center_of_mass(numpy.ones_like(labels), labels=labels)
```

Comment:
If we use `scipy.ndimage.measurements.center_of_mass`, you can save the Mahotas import.